playground/PIR with server/PIR+Server_Working.ino

```diff
@@ -0,0 +1,148 @@
+#include "config.h"
+#include <HSJsonConnector.h>
+
+// 1 Minute
+#define HANDSHAKE_INTERVAL 60000
+// 10 seconds
+#define DATA_INTERVAL 10000
+//Sensor ID
+// #define SENSOR_ID 500
+//Sensor type
+// #define SENSOR_TYPE "PIR"
+//Auth string
+#define SERVER_AUTH "Basic aHNfdXNlcjpycGNzX2hzMjAxOQ=="
+//Server URL
+#define SERVER_URL "http://kinect.andrew.cmu.edu:8000/hs/events"
+//Wifi SSID
+#define WIFI_SSID "Call kar Password le"
+//Wifi password
+#define WIFI_PASS "s7208043031"
```

Comment:
For the macros, it would be neat to define them in a `config.h` file and include that header file here.